specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1543,38 +1543,38 @@ requires that all Jakarta EE products that include both a web container and
 an EJB container provide support for access to local enterprise beans
 from the web container.
 
-The Servlet specification is available at
-_http://jcp.org/en/jsr/detail?id=369_ .
+The Jakarta Servlet specification is available at
+_https://jakarta.ee/specifications/servlet/4.0.2_ .
 
-=== JavaServer Pages™ (JSP) 2.3 Requirements
+=== Jakarta Server Pages (JSP) 2.3 Requirements
 
 The JSP specification depends on and builds
 on the servlet framework. A Jakarta EE product must support the entire JSP
 specification.
 
-The JSP specification is available at
-_http://jcp.org/en/jsr/summary?id=245_ .
+The Jakarta Server Pages specification is available at
+_https://jakarta.ee/specifications/pages/2.3.4_ .
 
-=== Expression Language (EL) 3.0 Requirements
+=== Jakarta Expression Language  (EL) 3.0 Requirements
 
-The Expression Language specification was
-formerly a part of the JavaServer Pages specification. It was split off
+The Jakarta Expression Language specification was
+formerly a part of the Jakarta Server Pages specification. It was split off
 into its own specification so that it could be used independently of
 JavaServer Pages. A Jakarta EE product must support the Expression
 Language.
 
-The Expression Language specification is
-available at _http://jcp.org/en/jsr/detail?id=341_ .
+The Jakarta Expression Language specification is
+available at _https://jakarta.ee/specifications/expression-language/3.0.2_ .
 
-=== Jakarta™ Message Service (JMS) 2.0 Requirements
+=== Jakarta Messaging 2.0 Requirements
 
-A Java Message Service provider must be
+A Jakarta Messaging provider must be
 included in a Jakarta EE product that requires support for JMS. The JMS
 implementation must provide support for both JMS point-to-point and
 publish/subscribe messaging, and thus must make those facilities
 available using the _ConnectionFactory_ and _Destination_ APIs.
 
-The JMS specification defines several
+The Jakarta Messaging specification defines several
 interfaces intended for integration with an application server. A Jakarta
 EE product need not provide objects that implement these interfaces, and
 portable Jakarta EE applications must not use the following interfaces:
```

Comment:
on the next line (which I couldn't comment on), the "j" is out of place...  It should be _javax.jms, not "j _avax.jms"